ide-netbeans.html

```diff
@@ -0,0 +1,300 @@
+<h2>JavaFX <span class="JFX">11</span> and NetBeans</h2>
+
+<p>This section explains how to use Java <span class="JDK">11</span> and JavaFX <span class="JFX">11</span> from NetBeans. 
+    Version Apache NetBeans <a href="https://dist.apache.org/repos/dist/dev/incubator/netbeans/incubating-netbeans/incubating-10.0-vc2/incubating-netbeans-10.0-vc2-bin.zip">10.0 vc2</a> 
+    was used for the following screenshots.
+</p>
+
+<p>
+    Download an appropriate <a href="https://jdk.java.net/11/">JDK <span class="JDK">11</span></a> for your operating system. 
+    Make sure <kbd>JAVA_HOME</kbd> is properly set to the Java <span class="JDK">11</span> installation directory. 
+</p>
+
+<div class="alert alert-warning">
+    <strong>Note: </strong>
+    
+    You can set NetBeans 10 to run with this JDK, by editing the <kbd>etc/netbeans.conf</kbd> file 
+    under the NetBeans installation folder and setting the <kbd>netbeans_jdkhome</kbd> property: 
+    
+<pre><code>
+netbeans_jdkhome=/path/to/jdk-<span class="JDK">11</span>
+</code></pre>
+
+    Save and restart NetBeans.
+    
+</div>
+
+<p>
+    You can create a JavaFX <span class="JFX">11</span> project and use <a href="#IDE">IDE</a>,
+    <a href="#Maven">Maven</a> or <a href="#Gradle">Gradle</a> build tools.
+</p>
+
+<a name="IDE"></a><h3>IDE</h3>
+
+<p>
+    Follow these steps to create a JavaFX project and use the IDE tools to build it and run it.
+</p>
+
+<p>
+    Download the appropriate
+    <a target="_blank" href="https://gluonhq.com/products/javafx/">JavaFX SDK</a>
+    for your operating system and unzip it to a desired location, for instance
+    <kbd>/Users/your-user/Downloads/javafx-sdk-<span class="JFX">11</span></kbd>.
+</p>
+
+<p>
+    Create a new global <kbd>Library</kbd> under <kbd>Tools -> Libraries -> New Library</kbd>. 
+    
+    Name it <kbd>JavaFX11</kbd> and include the jars under the <kbd>lib</kbd> folder from JavaFX <span class="JFX">11</span>.
+    
+    <a href="images/ide/netbeans/ide/netbeans00.png" target="_blank"><img src="images/ide/netbeans/ide/netbeans00.png" alt="Create library"/></a>
+    
+    Make sure you don't add the <kbd>src.zip</kbd> file.   
+</p>
+
+<p>
+    <strong>1. Create a Java project</strong>
+    
+    <a href="images/ide/netbeans/ide/netbeans01.png" target="_blank"><img src="images/ide/netbeans/ide/netbeans01.png" alt="Create a Java project"/></a>
+   
+    Provide a name to the project, like <kbd>HelloFX</kbd>, and a location.
+    
+    An default project will be opened.
+</p>
+
+<div class="alert alert-warning">
```

Comment:
May be we should use `alert-danger` style class here?

Reply:
Also, I have been thinking if we should remove explicitly adding "Note","Warning" etc. and just let the color do its work?